.ci_support/linux_64_cross_target_platformlinux-64gcc_machinex86_64gcc_maj_ver13gcc_version11.4.0glibc_version2.12.yaml

```diff
@@ -15,11 +15,11 @@ docker_image:
 gcc_machine:
 - x86_64
 gcc_maj_ver:
-- '10'
+- '13'
 gcc_vendor:
 - conda
 gcc_version:
-- 10.4.0
+- 11.4.0
```

Comment:
Hm... Something went wrong here...

Reply:
No idea why a local rerender produced the wrong result, but at least the bot got it right 🤷 

If you want I can squash those commits.

Reply:
I saw incorrect rerenders when doing the 2.28 sysroots.